ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/advancedsettings/view/AdvancedSettingsView.kt

```diff
@@ -92,98 +131,37 @@ private fun AdvancedSettingsTopAppBar(
     onBack: () -> Unit,
     subTitleState: TopAppBarSubTitleState
 ) {
-    SmallTopAppBar(
-        subTitle =
+    ZashiSmallTopAppBar(
+        title = stringResource(id = R.string.advanced_settings_title),
+        subtitle =
             when (subTitleState) {
                 TopAppBarSubTitleState.Disconnected -> stringResource(id = R.string.disconnected_label)
                 TopAppBarSubTitleState.Restoring -> stringResource(id = R.string.restoring_wallet_label)
                 TopAppBarSubTitleState.None -> null
             },
-        modifier = Modifier.testTag(AdvancedSettingsTag.ADVANCED_SETTINGS_TOP_APP_BAR),
+        modifier = Modifier.testTag(SettingsTag.SETTINGS_TOP_APP_BAR),
         showTitleLogo = true,
         navigationAction = {
-            TopAppBarBackNavigation(
-                backText = stringResource(id = R.string.back_navigation).uppercase(),
-                backContentDescriptionText = stringResource(R.string.back_navigation_content_description),
-                onBack = onBack
-            )
-        }
+            ZashiTopAppBarBackNavigation(onBack = onBack)
+        },
     )
 }
 
-@Suppress("LongParameterList")
+@Suppress("UnusedPrivateMember")
+@PreviewScreens
```

Comment:
We should improve the new `@PreviewScreens` annotation so we don't need to use `@Suppress("UnusedPrivateMember")` in every Compose preview. If we don't have it yet, let's please file a follow-up.

Reply:
done